File: zhiyoufy-server-java/zhiyoufy-server/src/main/java/com/example/zhiyoufy/server/domain/dto/wms/WmsActiveWorkerBase.java
package com.example.zhiyoufy.server.domain.dto.wms;

import java.util.Date;
import java.util.Map;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class WmsActiveWorkerBase {
	private boolean disconnected;
	private Date connectTime;
	private Date disconnectTime;
	private String sessionId;

	private String appRunId;
	private Date appStartTimestamp;

	private String workerName;

	private int maxActiveJobNum;
	private int freeActiveJobNum;
}

File: zhiyoufy-web/src/model/dto/wms/WmsActiveWorkerBase.ts
export interface WmsActiveWorkerBase {
  disconnected: boolean;
  connectTime: string;
  disconnectTime: string;
  sessionId: string;
  appRunId: string;
  appStartTimestamp: string;
  workerName: string;
  workerLabels: Record<string, string>;
  maxActiveJobNum: number;
  freeActiveJobNum: number;
}
